.gitignore

```diff
@@ -166,3 +166,5 @@ cython_debug/
 fix/
 workspace/
 results/
+*.ipynb

```

Comment:
If you want to exclude all notebooks, use `**/*.ipynb*` as this will remove `*.ipynb_checkpoints` also